### ISSUE TITLE
Undefined index: button_priority and other errors with WP 5.5 RC3 multisite

### ISSUE THREAD

**Issue by oldrup** [soflyy/oxygen-bugs-and-features#1407]
Hey. On a WordPress 5.5 RC 3 multisite running PHP 7.4 and with wp_debug true, this error is thrown when a Oxygen Builder 3.4 (stable) is enabled: 

`
Notice: Undefined index: button_priority in /home/oldrupdk/oldrup.net/wp-content/plugins/oxygen/component-framework/api/oxygen.element.class.php on line 38

Warning: Cannot modify header information - headers already sent by (output started at /home/oldrupdk/oldrup.net/wp-content/plugins/oxygen/component-framework/api/oxygen.element.class.php:38) in /home/oldrupdk/oldrup.net/wp-includes/functions.php on line 6270

Warning: Cannot modify header information - headers already sent by (output started at /home/oldrupdk/oldrup.net/wp-content/plugins/oxygen/component-framework/api/oxygen.element.class.php:38) in /home/oldrupdk/oldrup.net/wp-includes/pluggable.php on line 1296

Warning: Cannot modify header information - headers already sent by (output started at /home/oldrupdk/oldrup.net/wp-content/plugins/oxygen/component-framework/api/oxygen.element.class.php:38) in /home/oldrupdk/oldrup.net/wp-includes/pluggable.php on line 1299`

Only WordFence is also activated (network).

Let me know if there is anything I can do to troubleshoot this further.

Stay safe
Bjarne

**Comment by kehyian** [comment]
Hi there!
Actually I think is quite easy to fix that.
Just edit oxygen.element.class.php and add in line 16 the missing variable $button_priority.
Now the line 16 should look like this:
`function __construct($title, $id, $file='', $icon='', $button_place='', $button_priority='' , $other_options=array(), $has_js = false) {`

**Comment by oldrup** [comment]
Hey.. Just an update. WordPress has since been released stable version 5.5, and Oxygen stable 3.5, and I was kinda hoping I would not run into these errors any more. However, on my fresh multisite installation, these errors are still thrown. Is anyone able to reproduce this, or is it just me? 

Screenshot: https://i.postimg.cc/RFMd4BjQ/image.png

Kind regards
Bjarne

**Comment by Paciente8159** [comment]
> Hey.. Just an update. WordPress has since been released stable version 5.5, and Oxygen stable 3.5, and I was kinda hoping I would not run into these errors any more. However, on my fresh multisite installation, these errors are still thrown. Is anyone able to reproduce this, or is it just me?
> 
> Screenshot: https://i.postimg.cc/RFMd4BjQ/image.png
> 
> Kind regards
> Bjarne

I just bought my license...literally like minutes ago...and boom!!!
Same here....not a good start 
![image](https://user-images.githubusercontent.com/12116536/92622677-3b8c1b80-f2bd-11ea-967e-e66f98939825.png)

**Comment by Paciente8159** [comment]
by the way suppressed the error by replacing line 38 in oxygen.element.class.php from
`$this->params['button_priority']= $other_options['button_priority'];`
to
`if(array_key_exists('button_priority',$other_options))
		{
			$this->params['button_priority']= $other_options['button_priority'];
		}`

I want a discount :-DDDDD

**Comment by Paciente8159** [comment]
I've digged a bit more into the code and I think replacing it by:

```
if(isset($other_options['button_priority']))
{
    $this->params['button_priority']= $other_options['button_priority'];
}
```
is a better solution (in case `$other_options` is also NULL or undefined)
This error is triggered by the `Oxygen_Shape_Divider` class that extends `OxyEl` class. `Oxygen_Shape_Divider` does not implements the _button_priority_ method and that way the $other_options['button_priority'] key/value is not defined.

**Comment by KittenCodes** [comment]
Undefined index: button_priority fixed in v3.6.
